pkg/collector/horizontalpodautoscaler.go

```diff
@@ -78,6 +78,16 @@ func HorizontalPodAutoscaler(cfg config.Config, logger logr.Logger, otelcol v1al
 		}
 		metrics = append(metrics, targetCPUUtilization)
 
+		minReplicas := otelcol.Spec.Autoscaler.MinReplicas
+		if minReplicas == nil {
+			minReplicas = otelcol.Spec.MinReplicas
```

Comment:
Is it possible for the upgrade to call the validating webhook to ensure the user has a valid config especially after such a major difference upgrading from `v0.63.1` to `v0.78.0`?